pages/index.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>index</title>
+  <link href="/style/about.css" rel="stylesheet"> 
```

Comment:
Absolute paths can work well if your pages are in a variety of directories and you don't want to update copy/pasted code for styles/links. But we do need to make sure the root of our deployed server will match. There are two different ways to deploy to github pages. You would want to follow the User/Org instructions [here](https://pages.github.com/) instead of the project site deployment approach. Otherwise, you may want to use relative paths.